Woof/WoofTests/Helpers/Data Transformer Tests/ConvertValueZeroToFiveStyleTests.swift

```diff
@@ -0,0 +1,27 @@
+import XCTest
+
+final class ConvertValueZeroToFiveStyleTests: XCTestCase {
+    func testConvertValueReturnsZeroWhenNegativeValueProvided() {
+        XCTAssertEqual(DataTransformer.convertValueZeroToFiveStyle(-25.0), 0)
+    }
+
+    func testConvertValueReturnsZeroWhenZeroValueProvided() {
+        XCTAssertEqual(DataTransformer.convertValueZeroToFiveStyle(0), 0)
+    }
+
+    func testConvertValueReturnsOneWhenOneHalfValueProvided() {
+        XCTAssertEqual(DataTransformer.convertValueZeroToFiveStyle(0.5), 1)
+    }
+
+    func testConvertValueReturnsOneWhenValueBetweenOneHalfAndOneProvided() {
+        XCTAssertEqual(DataTransformer.convertValueZeroToFiveStyle(0.99), 1)
+    }
+
+    func testConvertValueReturnsOneWhenValueBetweenFourAndHalfAndFiveProvided() {
+        XCTAssertEqual(DataTransformer.convertValueZeroToFiveStyle(4.56), 5)
```

Comment:
Nit:
In the previous test you've already checked the rounding up. Maybe it makes sense to check the rounding down, for example:
```suggestion
        XCTAssertEqual(DataTransformer.convertValueZeroToFiveStyle(4.26), 5)
```

Reply:
Thanks, added more test cases to check the rounding.